### ISSUE TITLE
Upload from outside EPFL

### ISSUE THREAD

**Issue by kjappelbaum** [cheminfo/eln.epfl.ch#22]
Would be great to be able to do a `CURL` to the import folder for 

- [ ] XRD: `brml`  (Bruker) and `xy` (PowDLL)
- [ ] TGA: `txt` (TA Discovery)

in development: 

- [ ] Breakthrough `csv` (https://github.com/cheminfo/breakthrough-analysis/tree/master/testFiles) 
- [ ] Isotherm `dat` (DVS vacuum), `txt` (Micrometrics)

**Comment by lpatiny** [comment]
Are smb share folder working currently for importation ? There is a way to drop files from outside in those folders if the users have a gaspar account using the gaspar_product_batch filename format

**Comment by kjappelbaum** [comment]
>There is a way to drop files from outside in those folders if the users have a gaspar account using the gaspar_product_batch filename format

yes, this is what I meant. It is for some collaborators who have EPFL guest accounts

**Comment by kjappelbaum** [comment]
@lpatinywe should discuss this next Tuesday or some other point when you have time.

**Comment by lpatiny** [comment]
We would need:
- code for the instrument : room + kind (gcms6310 -> gc/ms in room BCH 5310)
- instrument : brand, model, software name + version

**Comment by kjappelbaum** [comment]
so, I got the information and the most relevant ones to start with are. 'hw' in the codes is for Heriot-Watt university  

gcmshw0151:
- Agilent, 7890B GC B.02.03.2, Openlab CDS Chemstation in vA.02.10[26]

dvshw0120:
- Surface Measurements Systems, DVS Vacuum, DVS Vacuum Control v2.2.4.0

irhw0121:
- Perkin Elmer, FTIR spectrometer frontier, Perkin Elmer Spectrum v10.03.06

tgahw0122: 
- TA instruments, TGA 550, Trios software in unknown version 

tgahw0123:
- TA Instruments, TGA Q500, QAdv.exe 2.9.396

xrdhw110: 
- Bruker, D8 Advance, Software and version unknown 


@lpatiny would be great if you can make the shares and I'll take care of the import.js scripts and tests.

**Comment by kjappelbaum** [comment]
For provenance, this is the full list of instruments i got 
[hw_instruments.txt](https://github.com/cheminfo/eln.epfl.ch/files/5937400/hw_instruments.txt)

For now i only selected those for which we have working parsers.

**Comment by lpatiny** [comment]
The following folders have been created in the folder '/cheminfo-submit/hw/ '

dvshw0120
gcmshw0151
irhw0121
tgahw0122
tgahw0123
xrdhw110

**Comment by kjappelbaum** [comment]
many thanks!
could i also ask for a `aiidalab01` folder?

**Comment by kjappelbaum** [comment]
More filetype info (email from march 1st)  

Hi Kevin,

—gcms0151: do you create cdf file? The file extension are .ch I asked Matthew to upload examples in the dropbox.


—mfc0150/mfc0130/mfc0132/cor0133: is the one that makes the txt files that are in the dropbox: Yes, the txt files in the folder: ELN_example_files\Breakthrough\Mass flow controller-Measurement

—ir0121: do you save jdx files? In the folder you can find an example of the raw data of the FTIR (.sp format). I was informed by the group that it is possible to export the file to excel and have the data in excel format. We think that it is better to upload in the ELN directly the sp. If it is not possible then we go for the excel

—mshp0131: ? I have not put files but I can tell you that is csv format. I add in the folder an example

—tem1010/sem1011: i assume you save TIF? I discussed with the Manohara to confirm that it is TIF as It is said in the ELN course. He uses TIF and we agree to confirm with everyone in the group that they have to use TIF.


Regards

E
